api/Cargo.toml

```diff
@@ -9,21 +9,21 @@ links = "deqs-api"
 [dependencies]
 deqs-quote-book-api = { path = "../quote-book/api" }
 
+futures = "0.3"
+grpcio = "0.12.0"
+
 mc-api = { path = "../mobilecoin/api" }
 mc-transaction-extra = { path = "../mobilecoin/transaction/extra" }
 mc-transaction-types = { path = "../mobilecoin/transaction/types" }
 mc-util-uri = { path = "../mobilecoin/util/uri" }
-
-futures = "0.3"
-grpcio = "0.12.0"
 protobuf = "2.27.1"
 rand = "0.8" # TODO something needs to enable the default features on rand, and since we don't have any dependency of mc-api doing that we need to do it here for now
 
 [dev-dependencies]
 deqs-mc-test-utils  = { path = "../mc-test-utils" }
 
 [build-dependencies]
-mc-util-build-grpc = { path = "../mobilecoin/util/build/grpc" }
-mc-util-build-script = { path = "../mobilecoin/util/build/script" }
 
 cargo-emit = "0.2.1"
+mc-util-build-grpc = { path = "../mobilecoin/util/build/grpc" }
```

Comment:
Why did this move?
The right way to run `cargo sort` is `cargo sort --workspace --grouped`, it looks like you might've ran it differently

Reply:
Thank you. I was missing the grouped part of that. Sorry about that.